dataloader/dataloader_constants.py

```diff
@@ -20,19 +20,35 @@
 COLLECTION_MENU_CATEGORIES = "menu_categories"
 COLLECTION_FILES_PARALLEL_COUNT = "files_parallel_count"
 COLLECTION_CATEGORIES_PARALLEL_COUNT = "categories_parallel_count"
-COLLECTION_SEARCH_INDEX = "search_index"
+COLLECTION_SEARCH_INDEX_TIB = "search_index_tib"
+COLLECTION_SEARCH_INDEX_SKT_PLI = "search_index_skt_pli"
 COLLECTION_SEARCH_INDEX_CHN = "search_index_chn"
-
-VIEW_SEARCH_INDEX = "search_index_view"
+VIEW_SEARCH_INDEX_TIB = "search_index_tib_view"
+VIEW_SEARCH_INDEX_TIB_FUZZY = "search_index_tib_fuzzy_view"
+VIEW_SEARCH_INDEX_SKT_PLI = "search_index_skt_pli_view"
 VIEW_SEARCH_INDEX_CHN = "search_index_chn_view"
+TIBETAN_ANALYZER = "tibetan_analyzer"
+TIBETAN_FUZZY_ANALYZER = "tibetan_fuzzy_analyzer"
+SANSKRIT_PALI_ANALYZER = "sanskrit_analyzer"
+CHINESE_ANALYZER = "text_zh"
```

Comment:
should this be `chinese_analyzer` like the other strings?

Reply:
text_zh is one of the builtin analyzers of arangodb, while the previous three ones defined here are custom analyzers. That's why the naming scheme is not identical... For our Chinese data the default is just fine!